Example/Optik/ViewController.swift

```diff
@@ -34,9 +62,24 @@ internal final class ViewController: UIViewController {
         let urls = [url1, url2, url3, url4]
         let imageDownloader = AlamofireImageDownloader()
         
-        let viewController = Optik.imageViewerWithURLs(urls, imageDownloader: imageDownloader)
+        let viewController = Optik.imageViewer(withURLs: urls, imageDownloader: imageDownloader)
```

Comment:
Why you don't call it with a currentImageIndex and set itself as delegate like for local images?